History.md

```diff
@@ -1,3 +1,43 @@
+Version 3.2.0
+=============
+
+Compatibility
+-------------
+
+* Ruby >= 2.7
+  - Support for Ruby 2.6 has been removed. Note that Ruby 2.6 reached
+    end-of-life in 2022-04.
```

Comment:
I think here is "reached the end-of-life" with "the".

Reply:
You're right. "reached the end-of-life" is indeed 5x more popular on Google.

Reply:
In my impression of English, if something is a proper noun, we shouldn't need to use "a" or "the". But otherwise, we may need those ones.

Reply:
I've added the missing "the".